src/patternfly/components/Pagination/examples/Pagination.md

```diff
@@ -23,14 +23,31 @@ cssPrefix: pf-c-pagination
 {{/pagination}}
 ```
 
-### Bottom
+### Top sticky
+```hbs isFullscreen
+{{#> pagination pagination--modifier="pf-m-sticky"}}
+  {{> pagination-total-items-content}}
+  {{> pagination-options-menu options-menu id="pagination-options-menu-top-example" options-menu--IsText="true"}}
+  {{> pagination-nav-content}}
+{{/pagination}}
+```
+
+### Bottom default
```

Comment:
I would probably just call this "Bottom" like it was before, and it will match the "Top" example.